utils.py

```diff
@@ -246,3 +249,16 @@ def check_software_installed(list_software=['sct']):
             logging.error("'{}' is not installed. Please install it before using this program.".format(software))
             install_ok = False
     return install_ok
+
+
+def create_empty_mask(fname, fname_label):
+    """
+    Create empty mask from reference image
+    :param fname: path to reference image
```

Comment:
Is this the absolute path? If so, then explicitly saying it as "absolute path to reference image" would be clear

Reply:
Good point! Addressed in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/b1f341c420f073350ad2bd45184af8a6f6e03785